eng/common/scripts/Typespec-Project-Generate.ps1

```diff
@@ -55,17 +55,17 @@ function NpmInstallForProject([string]$workingDirectory) {
 
 $resolvedProjectDirectory = Resolve-Path $ProjectDirectory
 $emitterName = &$GetEmitterNameFn
-$cadlConfigurationFile = Resolve-Path "$ProjectDirectory/cadl-location.yaml"
+$typespecConfigurationFile = Resolve-Path "$ProjectDirectory/typespec-location.yaml"
```

Comment:
Changes such as this will be dependent on getting all the repo's updated to use this new file.

Reply:
I'm not sure about the details of this tool, but I know for some tools like "Cadl Validation", we are planning to support both "cadl" and "typespec" names and file extensions until everything has been converted.

Reply:
That might be an interesting approach here as well.

Reply:
Yes. we can do this.